lib/neo4j/tasks/migration.rake

```diff
@@ -6,7 +6,12 @@ if !defined?(Rails) && !Rake::Task.task_defined?('environment')
   task :environment do
     require 'neo4j/session_manager'
     require 'ostruct'
-    Neo4j::Railtie.setup!
+    neo4j_url = ENV['NEO4J_URL'] || 'http://localhost:7474'
+    $:.unshift File.dirname('./')
+    Neo4j::ActiveBase.on_establish_session do
+      type = neo4j_url.match(/^bolt/) ? :bolt : http
```

Comment:
I think http here should be :http ..  
This commit resolves my issues if I set that environment variable to a bolt:// URL in the Rakefile but fails with this otherwise:
'NameError: undefined local variable or method `http' for main:Object'